### ISSUE TITLE
GPN-SS pre-train process

### ISSUE THREAD

**Issue by Ssupercoder** [songlab-cal/gpn#29]
Can you tell me how to finish pre-ptrain process of GPN-SS.Do I need to use four other files(such as: train.py).

**Comment by gonzalobenegas** [comment]
Hi! You only need `python -m gpn.ss.run_mlm`. In the past we trained for 120k steps with a constant learning rate and then 30k with a decaying learning rate. This could help as a guideline but depends on the amount of training data, model size, etc.
